### DISCUSSION TITLE
Rendering html code generated by a Python script

### DISCUSSION THREAD

**Discussion by lukefullard** [quarto-dev/quarto-cli#2654] [Q&A] [answered]
Hello Quarto community.

I am trying to create an html document which calls a Python script. The Python script generates some raw html, and then I want quarto to render the html.

1) I know I can run a Python script in Quarto like so:
```
'''{python}
import numpy as np
raw_html = myscript()
'''
```

2) I know I can render raw html like so:
```
'''{=html}
<center>
  Yay, html!
</center>
'''
```

But how can I render raw html **which is generated in the Python code block**?

Thanks for your help!

**Comment by mcanouil** [comment, marked as answer]
Using the yaml option `output: asis` as described in <https://quarto.org/docs/reference/cells/cells-jupyter.html#cell-output>. 😉

**Reply by lukefullard** [reply]
Ah legend! Thank you!
